### ISSUE TITLE
Pytorch CrossEntropy function contains softmax

### ISSUE THREAD

**Issue by youkaichao** [shaoxiongji/federated-learning#9]
Hi, thanks for your nice code.

However, I find that your code has a bug: you apply CrossEntropy function after softmax activation. But actually pytorch CrossEntropy function itself takes logit as its input.

After removing the softmax activation, I'm able to improve the MLP from 90% to 95%.

**Comment by youkaichao** [comment]
Then by changing the network architecture to the MLP (784 -> 200 -> 200 -> 10), the accuracy is increased to 97% :)

**Comment by shaoxiongji** [comment]
Thx again! fixed.
